src/components/ThemeProvider/tests/ThemeProvider.test.tsx

```diff
@@ -172,4 +173,410 @@ describe('<ThemeProvider />', () => {
       return null;
     }
   });
+
+  it('sets overrides', () => {
+    const wrapper = mountWithGlobalTheming(
+      <ThemeProvider theme={{}}>
+        <p>Hello</p>
+      </ThemeProvider>,
+      {globalTheming: true},
+    );
+
+    const styleKeys = Object.keys(wrapper.find('div')!.props.style || {});
```

Comment:
Update all these new tests to use the `toHaveReactProps` style assertions, e.g.

```ts
expect(themeProvider.find('div')).toHaveReactProps({
  style: expect.objectContaining({
    '--p-override-zero': '0',
  }),
});
```


We should probably collapse them all into one or two tests by putting multiple items in that objectContaining bit, that would make this less boilerplatey